templates/taxi/manufacturer_list.html

```diff
@@ -7,6 +8,10 @@ <h1>
       Create
     </a>
   </h1>
+  <form action="" method="get" class="form-inline">
+    {{ search_form|crispy }}
+    <input class="btn btn-secondary" type="submit" value="🌏">
+  </form>
```

Comment:
Create separate template and just include in in all places instead of duplicate code